Server/mods/deathmatch/logic/net/CSimVehiclePuresyncPacket.h

```diff
@@ -103,5 +103,7 @@ class CSimVehiclePuresyncPacket : public CSimPacket
         float fRailSpeed;
 
         SFixedArray<float, 4> fDoorOpenRatio;
+
+        bool isOnFire;
```

Comment:
Please, initialize it with the default value.

Reply:
Done